### DISCUSSION TITLE
Fast navigation for single sub folders

### DISCUSSION THREAD

**Discussion by afiffon** [community/community#3442] [Mobile]
Hi, I noticed that by using the mobile app I'm struggling to navigate through a number of single sub folders structure - I have to click the first and the second and so on, not like by using the browser where I can just click the first folder and it will navigate to the first folder with "split".
It really disrupts when opening a Java project, which usually has the structure of org/java/com/company/project.
Thanks

**Comment by PranamBhat** [comment]
@avivnaaman Thank you bringing this out! @evi-liu Can you please have a look here!

**Reply by bvalero-umh** [reply]
It seems the same than #55209
